### ISSUE TITLE
Raspberry Pi 4 Support

### ISSUE THREAD

**Issue by dgoodwin430** [sharky5102/fbmatrix#3]
Is this supported on the Raspberry Pi 4? I have not been able to get it to work. Maybe I am missing something.

**Comment by sharky5102** [comment]
Yes, in fact it should specifically work only in the pi4. What's the problem?

**Comment by dgoodwin430** [comment]
Thank you for your reply! I have been able to get it to output through the Matrix Bonnet but can not get it to display correctly on the LED Panel. I have tried three different panels. 32x32, 32x64 and 64x64. Just single panels, I have not done any chaining yet. My end goal is to use the 64x64 panel. I think the configuration for the type of panel is wrong but I am not sure where to change those settings. Is that the dpi_output_format or something else?
